courses/beginner/10.md

```diff
@@ -12,30 +12,23 @@ In addition to its own controls, an InSpec profile can bring in the controls fro
 
 When a profile includes controls from another profile, it is usually referred to as a “meta profile” or a “profile overlay.” Those of us with Chef background sometimes call it a “wrapper profile”.
 
-::: info Further information on inheritance
-You can find out more about inhertiance here [https://www.inspec.io/docs/reference/profiles/#sts=Profile%20Dependencies](https://www.inspec.io/docs/reference/profiles/#sts=Profile%20Dependencies)
-:::
-
-### Defining the Profile Dependency  
-
-Before a profile (e.g. `Profile A`) can use controls from another profile (e.g. `Profile B`), `Profile B` needs to be included in `Profile A`'s `inspec.yml` file in the `depends` section. Each profile that `Profile A` depends on should list its name and location. For example:
-
-```yaml
-depends:
- - name: linux-baseline
-   url: https://github.com/dev-sec/linux-baseline/archive/master.tar.gz
- - name: ssh-baseline
-   url: https://github.com/dev-sec/ssh-baseline/archive/master.tar.gz
+```mermaid
+flowchart LR
+    subgraph id1 [Profile]
+    my_nginx_overlay
+    end
+    subgraph id2 [Profile]
+    my_nginx-->my_nginx_overlay
+    end
 ```
 
-Once defined in the `inspec.yml` file, controls from the included profiles can be used!
-
-### Including All Controls from a Profile
+::: details Initializing the overlay
```

Comment:
Can you move this section to below the part about "To recap..." And make it not a drop down so people see that they need to do this step? You may add a sentence like:

Let's initialize an overlay profile that will use information from `my_nginx` profile.

Reply:
Added more context and moved thins around for a better flow. Thanks!